julia-mode.el

```diff
@@ -109,10 +109,17 @@
             (syntax open-parenthesis)
             (syntax whitespace)
             bol)
-        (group "'")
-        (or (repeat 0 8 (not (any "'"))) (not (any "\\"))
-            "\\\\")
-        (group "'"))))
+        (group "'")        ; start single quote of character constant
+        (or                ; two alternatives
+         (not (any "\\'")) ; one character, not single quote or backslash
```

Comment:
`'''` is a valid Char constant for `'\''`. This doesn't match that does it?

Reply:
Amazing, that must have been the only quoted single quote that the original code recognized, and the only one that's not documented in the julia manual.

This last pull also tightens up the \x syntax, julia only permits two hex digits, C permits as many as you write though not guaranteeing what will happen to the excess..